electrum/tests/regtest/regtest.sh

```diff
@@ -83,10 +96,10 @@ if [[ $1 == "init" ]]; then
     # alice is funded, bob is listening
     if [[ $2 == "bob" ]]; then
         $bob setconfig --offline lightning_listen localhost:9735
-    else
-        echo "funding $2"
-        $bitcoin_cli sendtoaddress $($agent getunusedaddress -o) 1
+        $bob setconfig --offline use_swapserver true
     fi
+    echo "funding $2"
+    $bitcoin_cli sendtoaddress $($agent getunusedaddress -o) 1
```

Comment:
Giving bob 1 btc potentially affects all tests that rely on bob's balance.

Fixed in https://github.com/spesmilo/electrum/commit/d9d281338cef699b70e2e816e1724a34586ef778

Reply:
thanks!